src/backend/modules/user/user.service.js

```diff
@@ -9,6 +10,8 @@ class Service {
 
     async createOne(userDto) {
         const isUserExist = await this.repository.findByEmail(userDto.email);
+        const hashStr = bcrypt.genSaltSync(10);
```

Comment:
- You should define SALT_ROUND in .env file for future use
- Create a file *bcrypt.service.js* in the modules/auth/service folder, there you create a class that is responsible for doing things related to password validation.